README.md

```diff
@@ -1,5 +1,13 @@
+
+
+ 
+<div align="center">
+<img width="800"  alt="notification-center-912bb96e009fb3a69bafec23bcde00b0" src="https://github.com/winglang/wing/assets/66887028/6023ceab-060e-4b48-be98-00a546e8c724" alt-text="GIF of Wing's Clouds">
```

Comment:
I think the image should exist in the repo itself and the src points to it, like the demo.gif below

Reply:
Ok, that's no problem.